es/orm.rst

```diff
@@ -1,26 +1,108 @@
-Models
-######
+Acceso a la base de datos & ORM
+###############################
+
+En CakePHP el acceso a la base de datos se hace por medio de dos objetos primarios. 
+El primero son **repositories** -repositorios- o **table objects** -objetos de tabla-. 
+Estos objetos proveen acceso a colecciones de datos. Nos permiten guardar nuevos 
+registros, modificar y borrar existentes, definir relaciones y realizar operaciones en masa.
+El segundo tipo de objeto son **entities** -entidades-. Las Entidades representan registros 
+individuales y permiten definir funcionalidad y comportamiento a nivel de registro/fila.
+
+Estas dos clases son responsables de manejar todo lo que sucede con datos, validez, interacción y 
+evolución en tu área de trabajo.
+
+El ORM incluído en CakePHP se especializa en base de datos relacionales, pero puede ser extendido 
+para soportar alternativas.
+
+El ORM de CakePHP toma ideas y conceptos de los modelos ActiveRecord y Datamapper. Aspira a 
+crear una implementación híbrida que combine aspectos de los dos modelos para crear un ORM rápido 
+y fácil de usar.
+
+Antes de comentar explorando el ORM, asegurate de configurar tu conexion :ref:`configure your
+database connections <database-configuration>`.
```

Comment:
It refers to a label database-configuration that does not yet exists in es files.
Please just add the following paragraph in [this file](https://github.com/cakephp/docs/blob/3.0/es/orm/database-basics.rst) after the ``..note``
```
.. _database-configuration:

Configuration
=============
```
This should stop the sphinx warning